build.sbt

```diff
@@ -61,6 +61,21 @@ lazy val formatSettings = SbtScalariform.scalariformSettings ++ Seq(
     .setPreference(SpacesWithinPatternBinders, true)
 )
 
+lazy val integrationTestArgument: Tests.Argument = {
```

Comment:
Please do this in the test, not the build. You can use scalatest itself to cancel the test with an appropriate message.